src/zenml/services/service.py

```diff
@@ -174,6 +174,23 @@ def check_status(self) -> Tuple[ServiceState, str]:
             service status).
         """
 
+    @abstractmethod
```

Comment:
No request to change anything, but making sure that this was intended as a non-optional method that any service needs to implement?

Reply:
Yes, I would consider this to be part of the service abstraction. If it cannot be implemented, then the subclass can just throw an exception or log a message saying so.